lib/checks/generic/page-no-duplicate-evaluate.js

```diff
@@ -18,7 +18,7 @@ function pageNoDuplicateEvaluate(node, options, virtualNode) {
   cache.set(key, true);
 
   let elms = querySelectorAllFilter(axe._tree[0], options.selector, elm =>
-    isVisible(elm.actualNode)
+    isVisible(elm.actualNode, true)
```

Comment:
I think this is now going to fail the scenario where there is a `main` with aria-hidden="true". I don't think that should fail.